src/server/network.ts

```diff
@@ -101,8 +100,6 @@ export class Request extends SdkObject {
     url: string, resourceType: string, method: string, postData: Buffer | null, headers: types.HeadersArray) {
     super(frame, 'request');
     assert(!url.startsWith('data:'), 'Data urls should not fire requests');
-    assert(!(routeDelegate && redirectedFrom), 'Should not be able to intercept redirects');
```

Comment:
Let's not pass routeDelegate anymore? I think it's null everywhere.

Reply:
done.